src/geojsonm/geojsonm.mli

```diff
@@ -79,3 +79,16 @@ val fold_geometry :
 val fold_props :
   ('a -> Ezjsonm.value -> 'a) -> 'a -> Jsonm.src -> ('a, Err.t) result
 (** [fold_props f init src] *)
+
+(** {2 Iterators}
+
+    Iterators works much more similar way as that of Maps. It takes an
+    imperative function of type 'a -> unit and an 'a list and applies the
+    function to every element in turn. The only difference is that Iterators
+    doesn't produce a new GeoJSON object (unlike mapping). *)
```

Comment:
This is copied from Lists I think so doesn't make a lot of sense in this context :)) 

```suggestion
     Iterators are similar to map functions except they a function [f] that takes a single
     element from the data-structure we are iterating over that must return [unit]. In that 
     sense they tend to functions with side-effects, such as [print_endline].
```

Something like that might be better.